tools/boards.txt.py

```diff
@@ -1573,7 +1573,7 @@ def usage (name,ret):
 # entry point
 
 lwip = 2
-default_speed = '115'
+default_speed = '921'
```

Comment:
I'm a little worried here about the speed change, see above comment.

Reply:
Speed change is now reverted.